scormxblock/scormxblock.py

```diff
@@ -308,7 +324,7 @@ def studio_submit(self, request, suffix=''):
             # Destroy temp directory after all files are copied.
             temp_directory.close()
 
-        return Response({'result': 'success'}, content_type='application/json')
+        return Response({'result': 'success'}, content_type='application/json', charset='utf8')
```

Comment:
Why is necessary specify the character encoding?

Reply:
Now the Reponse class requires to specify the charset otherwise it throws an error